tests/components/mqtt/test_tag.py

```diff
@@ -46,13 +45,6 @@
 )
 
 
-@pytest.fixture
-def tag_mock() -> Generator[AsyncMock]:
-    """Fixture to mock tag."""
-    with patch("homeassistant.components.tag.async_scan_tag") as mock_tag:
-        yield mock_tag
-
-
 @pytest.mark.no_fail_on_log_exception
```

Comment:
This was moved because the fixture was needed for some added discovery tests with device-based discovery